client/views/parameters-editor.js

```diff
@@ -88,6 +96,24 @@ module.exports = View.extend({
       });
     });
   },
+  renderViewParameter: function () {
+    if(this.viewParameterView) {
+      this.viewParameterView.remove();
+    }
+    this.containsMdlWithAnn = this.collection.filter(function (model) {return model.annotation}).length > 0;
+    if(!this.containsMdlWithAnn) {
+      $(this.queryByHook("parameters-annotation-header")).css("display", "none");
+    }else{
+      $(this.queryByHook("parameters-annotation-header")).css("display", "block");
+    }
+    let options = {viewOptions: {viewMode: true}};
+    this.viewParameterView = this.renderCollection(
+      this.collection,
+      EditParameterView,
+      this.queryByHook('view-parameter-list'),
+      options
+    );
+  },
   addParameter: function () {
     this.collection.addParameter();
     $(document).ready(function () {
```

Comment:
(Unrelated to PR) jQuery's `$(document).ready()` function appears to be deprecated

Reply:
fixed